packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -155,15 +157,62 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         // TODO: Mark msg.sender as already voted on electionVotes
     }
 
-    function _isNextEpochNomination() internal view virtual returns (bool) {
-        return false;
+    function epochFinised() external view override returns (bool) {
+        if (_electionStore().epochStart == 0) {
+            revert EpochNotStarted();
+        }
+
+        return _epochFinished();
```

Comment:
Is it really necessary to wrap this in an internal function? (Also applies in other parts of the file)